### ISSUE TITLE
Various bugfixes for LW2 1.5

### ISSUE THREAD

**Issue by Clibanarius** [long-war-2/lwotc#838]
So a few of these bugs fixed in faket15's unofficial bugfix recompile of LW2 1.5 are still present in LWotC. I'd like to see the changes imported over! Stuff like cone attack damage falloff working with SOURCE TILE instead of TARGET. And Liberation 3 being re-detected immediately upon spawn no matter what if you've detected it once before.

https://pavonisinteractive.com/phpBB3/viewtopic.php?f=18&t=27394

**Comment by robojumper** [comment]
It would be good if faket15 could make the source code changes available. While all config changes are there in plaintext, only compiled script files are included, and they would have to be decompiled and the changes reverse-engineered to import them over.

**Comment by Clibanarius** [comment]
Indeed, I didn't think to check if the source was included, as it is not. I can do my best to sift through the compiled script(s, although the Alien Pack and Officer Pack changes are probably minimal) and find the desired changes, if need be!

**Comment by Grobobobo** [comment]
Is there a way to contact him?

**Comment by Clibanarius** [comment]
PM on Pavonis Forums, he was active as of the 2nd, so he might see it at some point; I sent him a message.
